### ISSUE TITLE
When switching scenes, audible return level of scene A, does not reset to match displayed return level of scene B

### ISSUE THREAD

**Issue by KosmiKFead** [surge-synthesizer/surge#5873]
_**Bug Description:**_
When a the returns of scene A are linked to a scene mod, so that the returns are audibly heard modulating, and one switches to scene B, where the returns are at 0, the internal level of the returns do not go to the displayed value of scene B, and the returns are audible even with the return's displayed value being 0 on scene B.

On switching from scene A to B, the return stays, where ever the mod had it, leaving audible send reverb even when the return of B is at 0%. 

_**Surge XT Version**_
Version: Surge XT 1.1.nightly.f2d98f5
Build: 2022-02-08 @ 03:33:20 on 'fv-az33-554/pipeline' with 'MSVC-19.29.30139.0' using JUCE 6.1.2

**Reproduction Steps:**
1. Add reverb to scene A sends,
2. Add a scene A S-mod to the Returns so that its audibly fading the reverb in and out.
3. Make scene B's returns be 0 without mods,
4. Switch from scene A to B
5. hear the return level NOT going to the displayed level of 0.

**Expected Behavior:**
I expected the audible return level to go to 0 as scene B displays.

**Screenshots:**

https://user-images.githubusercontent.com/70804661/153028364-b301cf8b-2745-42eb-a3af-f792988b5dc5.mp4

**Computer Information (please complete the following!):**
 System: Windows 64-bit VST3 on Intel(R) Core(TM) i7-4770S CPU @ 3.10GHz
Host: FruityLoops 20.9 @ 48.0 kHz

**Additional Information:**
The Return's level of scene B is added to the "stuck" level of Scene A,

**Comment by mkruselj** [comment]
OK so maybe there's some confusion here. Return levels are not per scene, they are global parameters pertaining to send FX slots themselves.

**Comment by baconpaul** [comment]
[DemoReturnMod 5873.fxp.zip](https://github.com/surge-synthesizer/surge/files/8502436/DemoReturnMod.5873.fxp.zip)

OK this is now fixed. I think I fixed it when I fixed that index vs scene bug for j5v actually after 1.0.1

Attached is a patch to show its fixed though. This has two sends a distortion and a ring modulator.
Scene A sends to the distortion scene B to the modulator
Scene A SLFO opens the distortion send and scene B the modulator
and if you play in single you hear one or the other and in dual you hear both

So I think it was broken but it's clearly fixed now. So I'm going to close this issue.
